inc/Widget/SmailyWidget.php

```diff
@@ -88,10 +91,22 @@ public function widget( $args, $instance ) {
 
 		// Widget front-end.
 		// Echo messages if available.
-		if ( isset( $_GET['message'] ) ) {
+		if ( isset( $_GET['code'] ) ) {
+			switch ( $_GET['code'] ) {
+				case 101:
+					$message = __( 'Subscription registration successful.', 'smaily' );
+					break;
+				case 204:
+					$message = __( 'Subscription do not contain a valid email address.', 'smaily' );
```

Comment:
You should use `does` instead of `do`.
> “Do” is used when referring to two or more persons or things while “does” is used when referring to a single person or thing.

Reply:
Fixed in https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/43/commits/908114bfdd9b5c85ae83fb794fd137fc13ab4870